release-notes/major12/12.2.0/valtimo-backend-libraries.md

```diff
@@ -14,6 +14,11 @@ The following features were added:
   which caused several events and checks to trigger as a side effect.
   This could potentially affect performance. They now use repositories directly.
 
+* **Keycloak supports OAuth 2.0 authentication**
```

Comment:
This does not really describe what's changed. Valtimo always supported OAuth 2.0 via Keycloak. It was just never configured using the default Spring OAuth support.

Reply:
changed